src/App.jsx

```diff
@@ -14,45 +17,78 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_ALPHAB = 'name';
+const SORT_BY_LENGTH = 'length';
+
+export const App = () => {
+  let goods = [...goodsFromServer];
+  const [sortField, setSortField] = useState('');
+  const [isReverse, setIsReverse] = useState(false);
+
+  if (sortField === SORT_BY_ALPHAB) {
+    goods = goods.sort();
+  }
+
+  if (sortField === SORT_BY_LENGTH) {
```

Comment:
Consider using switch...case here as you have a limited amount of sortFields
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch
Also, please put this logic inside some function (e.g. getSortedGoods)